event.json

```diff
@@ -1,12 +0,0 @@
-{
-    "source": "aws.events", 
-    "account": "123456789012", 
-    "time": "1970-01-01T00:00:00Z", 
-    "id": "cdc73f9d-aea9-11e3-9d5a-835b769c0d9c", 
-    "region": "eu-central-1", 
-    "detail": {}, 
-    "resources": [
-        "arn:aws:events:us-east-1:123456789012:rule/my-schedule"
-    ], 
-    "detail-type": "Scheduled Event"
-}
```

Comment:
Is this a temp file being removed?

Reply:
Yep, this was a leftover from the initial lambda-based approach.